static/js/components/ObservationPlanRequestForm.jsx

```diff
@@ -47,6 +51,111 @@ const useStyles = makeStyles(() => ({
   },
 }));
 
+const ObservationPlanGlobe = ({ loc, instrument, setSelectedFields }) => {
+  const dispatch = useDispatch();
+
+  const displayOptions = [
+    "localization",
+    "sources",
+    "galaxies",
+    "instrument",
+    "observations",
+  ];
+  const displayOptionsDefault = Object.fromEntries(
+    displayOptions.map((x) => [x, false])
+  );
+  displayOptionsDefault.localization = true;
+  displayOptionsDefault.instrument = true;
+
+  const [skymapInstrument, setSkymapInstrument] = useState(null);
+  useEffect(() => {
+    const fetchSkymapInstrument = async () => {
+      const response = await dispatch(
+        GET(
+          `/api/instrument/${instrument.id}?includeGeoJSONSummary=True&localizationDateobs=${loc.dateobs}&localizationName=${loc.localization_name}`,
+          "skyportal/FETCH_INSTRUMENT"
+        )
```

Comment:
Wouldn't it be more kosher with the rest of the codebase to put this in a duck file? The long url seems out of place

Reply:
Agreed